tests/unit/plugins/test_ant_plugin.py

```diff
@@ -122,8 +122,9 @@ def test_get_build_snaps_and_packages(part_info):
 def test_get_build_environment(part_info):
     properties = AntPlugin.properties_class.unmarshal({"source": "."})
     plugin = AntPlugin(properties=properties, part_info=part_info)
-
-    assert plugin.get_build_environment() == {}
+    env = plugin.get_build_environment()
+    assert "JAVA_HOME" in env
+    assert len(env) == 1
```

Comment:
This is a bit fragile based on other changes we might make. Would it make more sense to have:

```suggestion
    assert "ANT_OPTS" not in env
```